app/models/seller.rb

```diff
@@ -40,6 +40,8 @@ class Seller < ApplicationRecord
   has_many :menu_items, dependent: :destroy
   has_many :items, dependent: :destroy
 
+  has_one :recipient, class_name: 'Contact'
```

Comment:
This could either use a better name or more documentation around what it's used for

maybe partner? distributor? buy_a_meal_for_partner? buy_a_meal_for_distributor?

Reply:
Done! Named it to distributor